pallets/schemas/README.md

```diff
@@ -0,0 +1,79 @@
+# Schemas Pallet
+
+The Schemas Pallet provides universal schema registration for data flowing through Frequency.
+
+## Summary
+
+This pallet provides an on chain repository for schemas, thereby allowing participants of the network to flexibly interact and exchange messages with each other with the correct human intent and data structure.
+All Messages and Stateful Storage content must be attached to a Schema Identifier so that the content can be correctly parsed and interpreted.
```

Comment:
This could use a link to the Messages and Stateful Storage docs, I guess when they are created. That, or it could link to the code I suppose.

Reply:
Hmm... I've been avoiding links that should be between these Readmes as I'm not yet sure how to handle those as they are getting used in two places (Rust docs and Docs).

I've added a task to the https://github.com/frequency-chain/docs/issues/59 issue to add crosslinks at the end.